tools/build/mrwebrtc/uwp/mrwebrtc-uwp.vcxproj

```diff
@@ -50,7 +50,7 @@
     <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>

```

Comment:
nit: any particular reason to move the vcxprojs to tools/build?

Reply:
A bit of consistency. The other build tools for Android and libwebrtc (undock) are already there.